clevercloud/crons/mailerlite-accueil-sync.sh

```
@@ -0,0 +1,10 @@
+#! /bin/bash -l
+
+# Important to keep this file
+# Clevercloud requires this to load environment
+# https://developers.clever-cloud.com/doc/administrate/cron/#access-environment-variables
+
+cd ${APP_HOME}
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)